### ISSUE TITLE
"bower install --save angular-ui-ace#bower"  do not add the all needed references to index.html that ace editor were working

### ISSUE THREAD

**Issue by vdrasutis** [angular-ui/ui-ace#136]
Hello I am using:
- angular-fullstack generator v3.2.0
- npm v 3.5.3
- node v4.2.4
- bower v1.7.7
- angular 1.4.9

When executin in cmd
`bower install --save angular-ui-ace#bower`
Bower adds only this
`<script src="bower_components/angular-ui-ace/ui-ace.js"></script>`
reference to index.html.

Is it should do not add (work like this) below written reference or this is a bug?
`<script type="text/javascript" src="bower_components/ace-builds/src-min-noconflict/ace.js"></script>`

**Comment by jvictorsoto** [comment]
You are right, bower.json should include it in main variable, this fix it:
`"main": ["./ui-ace.js", "../ace-builds/src-min-noconflict/ace.js"],`

**Comment by tihomir-kit** [comment]
Any idea when this might be patched?

**Comment by mmumshad** [comment]
+1

**Comment by jfyzhao** [comment]
+1

Is this repo still maintained at all?
